guides/keras_cv/classification_with_keras_cv.py

```diff
@@ -0,0 +1,701 @@
+"""
+
+train_ds = train_ds.ragged_batch(BATCH_SIZE)
+eval_ds = eval_ds.ragged_batch(BATCH_SIZE)
+
+batch = next(iter(train_ds.take(1)))
+image_batch = batch["images"]
+label_batch = batch["labels"]
+
+keras_cv.visualization.plot_image_gallery(
+    image_batch.to_tensor(),
+    rows=3,
+    cols=3,
+    value_range=(0, 255),
+    show=True,
+)
+"""
+
+## Data Augmentation
+
+In our previous finetuning exmaple, we performed a static resizing operation and
+did not include any image augmentation.
+This is because a single pass over the training set was sufficient to achieve
+decent results.
+When training to solve a more difficult task, you'll want to include data
+augmentation in your data pipeline.
+
+Data augmentation is a technique to make your model robust to changes in input
+data such as lighting, cropping, and orientation.
+KerasCV includes some of the most useful augmentations in the keras_cv.layers
+API.
+Creating an optimal pipeline of augmentations is an art, but in this section of
+the guide we'll offer some tips on best practices for classification.
+
+One caveat to be aware of with image data augmentation is that you must be careful
+to not shift your augmented data distribution too far from the original data
+distribution.
+The goal is to introduce noise to prevent overfitting and increase generalization,
+but samples that lie completely out of the data distribution simply add noise to
+the training process.
+
+The first augmentation we'll use is `RandomFlip`.
+This augmentation behaves more or less how you'd expect: it either flips the
+image or not.
+While this augmentation is useful in CalTech101 and ImageNet, it should be noted
+that it should not be used on tasks where the data distribution is not vertical
+mirror invariant.
+An example of a dataset where this occurs is MNIST hand written digits.
+Flipping a `6` over the
+vertical axis will make the digit appear more like a `7` than a `6`, but the
+label will still show a `6`.
+"""
+random_flip = keras_cv.layers.RandomFlip()
+augmenters = [random_flip]
+
+image_batch = random_flip(image_batch)
+keras_cv.visualization.plot_image_gallery(
+    image_batch.to_tensor(),
+    rows=3,
+    cols=3,
+    value_range=(0, 255),
+    show=True,
+)
+"""
+Half of the images have been flipped!
+
+The next augmentation we'll use is `RandomCropAndResize`.
+This operation selects a random subset of the image, then resizes it to the
+provided target size.
+By using this augmentation, we force our classifier to become spatially invariant.
+Additionally, this layer accepts an `aspect_ratio_factor` which can be used to
+distort the aspect ratio of the image.
+While this can improve model performance, it should be used with caution.
+It is very easy for an aspect ratio distortion to shift a sample too far from
+the original training set's data distribution.
+Remember - the goal of data augmentation is to produce more training samples
+that align with the data distribution of your training set!
```

Comment:
Not training set but your real world application! If you want the training distribution use the training data.

Even here I'm a little skeptical if this a a good summary. `CutMix` images look nothing like a real image. The real goal to actually to make the classifier robust to distribution shifts in the images at inference time.